### PR TITLE
This huge PR adds proper SEO to all articles in the docs for better search engine indexing

### PR DESCRIPTION
**Pull request by cryptobench**

### CODE REVIEW

File: src/pages/docs/creators/javascript/examples/accessing-internet.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Accessing the Internet
-title: Accessing the Internet
+title: Golem JS Task API - Internet Access via Outbound Feature
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/examples/accessing-internet.md:2]:
```suggestion
title: Golem JS Task API - Internet Access
```

File: src/pages/docs/creators/javascript/examples/accessing-internet.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+description: Learn to access the internet using Golem's JS Task API. Includes steps for manifest creation, demand definition, and running Node.js examples.
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/examples/accessing-internet.md:3]:
```suggestion
description: Learn to access the internet using Golem's JS Task API. Discusses the Outbound feature and includes steps for manifest creation, demand definition, and running Node.js examples.
```

File: src/pages/docs/creators/javascript/examples/index.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
-description: List of examples
-title: JS Examples
+description: Comprehensive guide to JavaScript code examples for Golem Network tasks, data transfer, provider selection, and more.
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/examples/index.md:2]:
```suggestion
description: Comprehensive list of JavaScript code examples for Golem Network tasks, data transfer, provider selection, and more.
```

File: src/pages/docs/creators/javascript/examples/composing-tasks.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Composing tasks
-title: Composing tasks
+description: Learn to compose tasks using JS Task API, featuring methods for sequential and batch command execution with practical examples.
+title: Composing and Executing Tasks
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/examples/composing-tasks.md:3]:
```suggestion
title: Composing and Organizing Tasks
```

File: src/pages/docs/creators/javascript/examples/executing-tasks.md [hunk 5]
```diff
@@ -1,6 +1,8 @@
 ---
-description: Executing task
-title: Executing task
+title: Executing Tasks - Examples and Methods
+pageTitle: Guide to Executing Tasks with Golem's JavaScript API
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/examples/executing-tasks.md:3]:
```suggestion
pageTitle: Executing Tasks with Golem's JavaScript API
```

File: src/pages/docs/creators/javascript/examples/switching-to-mainnet.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Switching to mainnet
-title: Switching to mainnet
+title: JS Task API Guide - Switching from Testnet to Mainnet on Golem Network
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/examples/switching-to-mainnet.md:2]:
```suggestion
title: JS Task API - Switching from Testnet to Mainnet in practical examples
```

File: src/pages/docs/creators/javascript/examples/switching-to-mainnet.md [hunk 7]
```diff
@@ -1,6 +1,7 @@
...
+description: Learn how to switch from testnet to mainnet for executing tasks on the Golem Network using the JS SDK, with a focus on requestors and the use of Polygon.
+pageTitle: Transition to Golem Mainnet for Enhanced Task Execution - A Guide for JS SDK Users
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/examples/switching-to-mainnet.md:4]:
```suggestion
pageTitle: Transition to Golem Mainnet for Enhanced Task Execution - Best Practices and Examples
```

File: src/pages/docs/creators/javascript/examples/using-app-keys.md [hunk 8]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Using app-keys
-title: Using app-keys
+description: Guide on using app-keys with JS Task API in Yagna REST API, including creating, listing, and integrating unique keys for secure access.
+title: Using app-keys 
+pageTitle: Secure Your Yagna REST API Access - JS Task API App-Key Integration Guide
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/examples/using-app-keys.md:4]:
```suggestion
pageTitle: Secure Your Yagna REST API Access - JS Task API App-Key Integration Examples
```

File: src/pages/docs/creators/javascript/examples/working-with-images.md [hunk 9]
```diff
@@ -1,6 +1,7 @@
 ---
-title: Working with Golem images
-description: Examples of how to use your own Golem image in a requestor script
+title: Guide to Utilizing Custom Golem Images in Requestor Scripts
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/examples/working-with-images.md:2]:
```suggestion
title: Examples of utilizing Custom Golem Images in Requestor Scripts
```

File: src/pages/docs/creators/javascript/guides/accessing-internet.md [hunk 10]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Learn how to access the internet from golem providers
-title: Accessing the internet
+title: Guide to Outbound Networking in Golem
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/guides/accessing-internet.md:2]:
```suggestion
title: Guide to accessing the Internet from Golem providers
```

File: src/pages/docs/creators/javascript/quickstarts/golem-in-a-browser.md [hunk 11]
```diff
@@ -1,6 +1,7 @@
 ---
-description: A minimal example of a functional Golem requestor agent in a browser
-title: Requestor in browser Quickstart
+description: Learn how to set up a Golem requestor agent in your browser, including Yagna service configuration and running scripts in a web context.
+title: Browser-Based Golem Requestor Agent Setup Guide
+pageTitle: Implementing Golem Requestor Agents in Web Browsers - A Step-by-Step Tutorial
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/quickstarts/golem-in-a-browser.md:4]:
```suggestion
pageTitle: Implementing Golem Requestor Agents in Web Browsers - Quickstart
```

File: src/pages/docs/creators/javascript/quickstarts/index.md [hunk 12]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Golem JS Quickstarts
+description: Table of contents of Golem JS Quickstarts
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/quickstarts/index.md:2]:
```suggestion
description: List of Golem JS Quickstarts
```

File: src/pages/docs/creators/javascript/quickstarts/quickstart.md [hunk 13]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Create your own JavaScript application on Golem
-title: Golem Node.js Quickstart
+title: Node.js Development on Golem Network
+pageTitle: Easily Build JavaScript Apps Using Golem Network - Node.js Quickstart Guide
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/quickstarts/quickstart.md:3]:
```suggestion
pageTitle: Easily Build JavaScript Apps Using Golem Network - Node.js Quickstart
```

File: src/pages/docs/creators/javascript/quickstarts/quickstart.md [hunk 14]
```diff
@@ -1,6 +1,7 @@
...
+description: Learn to build JavaScript applications with this quickstart guide for the Golem Network, featuring installation steps, code examples, and easy integration.
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/quickstarts/quickstart.md:4]:
```suggestion
description: Learn to build JavaScript applications with this quickstart for the Golem Network, featuring installation steps, code examples, and easy integration.
```

File: src/pages/docs/creators/javascript/quickstarts/retrievable-task.md [hunk 15]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Create a retrievable task on Golem
-title: Golem Retrievable Task (Job API) Quickstart
+title: Golem Network Job API - Executing and Retrieving Long-Running Tasks
+pageTitle: Efficiently Run and Retrieve Long-Duration Tasks on Golem Network - A Step-by-Step Quickstart Guide
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/quickstarts/retrievable-task.md:3]:
```suggestion
pageTitle: Efficiently Run and Retrieve Long-Duration Tasks on Golem Network - A Step-by-Step Quickstart
```

File: src/pages/docs/creators/javascript/react/getting-started.md [hunk 16]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Get started with the React SDK
-description: Get started with the React SDK
+title: React SDK Guide - Installation, Configuration, and Usage
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/react/getting-started.md:2]:
```suggestion
title: React SDK Introduction - Installation, Configuration, and Usage
```

File: src/pages/docs/creators/javascript/tutorials/accessing-internet.md [hunk 17]
```diff
@@ -1,6 +1,7 @@
 ---
-title: Tutorial on how to access the Internet from the Provider
-description: Accessing the Internet from Provider Tutorial
+title: Enabling Internet Access for Requestor Tasks on Golem Network
+description: Step-by-step guide for Golem requestors to enable internet access in tasks, detailing manifest creation and provider security.
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/tutorials/accessing-internet.md:3]:
```suggestion
description: Step-by-step tutorial for Golem requestors to enable internet access in tasks, detailing manifest creation and provider security.
```

File: src/pages/docs/creators/javascript/tutorials/accessing-internet.md [hunk 18]
```diff
@@ -1,6 +1,7 @@
...
+pageTitle: Golem Requestors' Guide to Internet Access for Tasks
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/tutorials/accessing-internet.md:4]:
```suggestion
pageTitle: Golem Requestors' Tutorial on how to access Internet for Tasks
```

File: src/pages/docs/creators/javascript/tutorials/building-custom-image.md [hunk 19]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Creating and using images on Golem
-description: Creating and using images on Golem
+title: Docker to Golem Image Conversion and Usage Guide
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/tutorials/building-custom-image.md:2]:
```suggestion
title: Docker to Golem Image Conversion and Usage Tutorial
```

File: src/pages/docs/creators/javascript/tutorials/index.md [hunk 20]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Tutorials
+description: Table of contents for the Golem JS SDK tutorials.
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/tutorials/index.md:2]:
```suggestion
description: List of the Golem JS SDK tutorials.
```

File: src/pages/docs/creators/javascript/tutorials/quickstart-explained.md [hunk 21]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Create your own JavaScript application on Golem
-title: Node.js Quickstart
+title: Node.js Application Development on Golem Network
+description: Learn to create a JavaScript application using the Golem Network with our comprehensive guide, featuring hands-on steps for setting up Node.js projects and utilizing Task Executors.
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/tutorials/quickstart-explained.md:3]:
```suggestion
description: Learn to create a JavaScript application using the Golem Network with our tutorial where we explain the Quickstart example and steps for setting up Node.js projects and utilizing Task Executors.
```

File: src/pages/docs/creators/javascript/tutorials/rest-api-with-express.md [hunk 22]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Step by step guide to create a REST API with Express.js that uses the Golem Network to process data.
-title: REST API example with Express.js
+title: Creating a Text-to-Speech API with Express.js and Golem Network
+pageTitle: Build a REST API for Text-to-Speech Conversion Using Express.js and Golem Network - Step-by-Step Tutorial
+description: Learn to build a REST API with Express.js utilizing the Golem Network for text-to-speech processing. A practical guide with code examples.
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/tutorials/rest-api-with-express.md:4]:
```suggestion
description: Learn to build a REST API with Express.js utilizing the Golem Network for text-to-speech processing. A practical tutorial.
```

File: src/pages/docs/creators/javascript/tutorials/running-in-browser.md [hunk 23]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Golem in browser Quickstart explained
+description: Step-by-step guide on integrating Golem network with a browser, covering installation, setup, and script execution.
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/tutorials/running-in-browser.md:2]:
```suggestion
description: Step-by-step tutorial on integrating Golem network with a browser, covering installation, setup, and script execution.
```

File: src/pages/docs/creators/python/examples/tools/converting-docker-image-to-golem-format.md [hunk 24]
```diff
@@ -1,6 +1,7 @@
 ---
-title: Converting an image
-description: Instructions on how to convert an image from Docker to Golem using gvmkit-build
+title: Docker to Golem Image Conversion Guide
```
Comment (jalas167) [inline src/pages/docs/creators/python/examples/tools/converting-docker-image-to-golem-format.md:2]:
```suggestion
title: Docker to Golem Image Conversion Example 
```

File: src/pages/docs/creators/python/examples/tools/converting-docker-image-to-golem-format.md [hunk 25]
```diff
@@ -1,6 +1,7 @@
 ---
-title: Converting an image
-description: Instructions on how to convert an image from Docker to Golem using gvmkit-build
+title: Docker to Golem Image Conversion Guide 
+description: Step-by-step guide for converting Docker images to Golem format using gvmkit-build, with prerequisites and examples included.
```
Comment (jalas167) [inline src/pages/docs/creators/python/examples/tools/converting-docker-image-to-golem-format.md:3]:
```suggestion
description: An article on how to convert Docker images to Golem format using gvmkit-build, with prerequisites and examples included. 
```

File: src/pages/docs/creators/python/examples/tools/converting-docker-image-to-golem-format.md [hunk 26]
```diff
@@ -1,6 +1,7 @@
 ---
-title: Converting an image
-description: Instructions on how to convert an image from Docker to Golem using gvmkit-build
+title: Docker to Golem Image Conversion Guide 
+description: Step-by-step guide for converting Docker images to Golem format using gvmkit-build, with prerequisites and examples included. 
+pageTitle: Convert Docker Images to Golem with gvmkit-build - Step-by-Step Guide
```
Comment (jalas167) [inline src/pages/docs/creators/python/examples/tools/converting-docker-image-to-golem-format.md:4]:
```suggestion
pageTitle: Convert Docker Images to Golem with gvmkit-build - Step-by-Step Example
```

File: src/pages/docs/creators/python/examples/tools/golem-wallet-backup.md [hunk 27]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Ensuring the safety of your Golem funds.
-title: Golem wallet backup
+title: Golem Wallet Backup Instructions 
+description: Step-by-step guide to securely back up your Golem wallet, ensuring the safety and accessibility of your GLM/ETH tokens.
```
Comment (jalas167) [inline src/pages/docs/creators/python/examples/tools/golem-wallet-backup.md:3]:
```suggestion
description: Step-by-step instructions on how to securely back up your Golem wallet, ensuring the safety and accessibility of your GLM/ETH tokens. 
```

File: src/pages/docs/creators/python/examples/tools/golem-wallet-backup.md [hunk 28]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Ensuring the safety of your Golem funds.
-title: Golem wallet backup
+title: Golem Wallet Backup Instructions 
+description: Step-by-step guide to securely back up your Golem wallet, ensuring the safety and accessibility of your GLM/ETH tokens. 
+pageTitle: Secure Your Golem Cryptocurrency - Wallet Backup Guide
```
Comment (jalas167) [inline src/pages/docs/creators/python/examples/tools/golem-wallet-backup.md:4]:
```suggestion
pageTitle: Secure Your Golem Cryptocurrency - Wallet Backup Instructions
```

File: src/pages/docs/creators/python/examples/tools/managing-golem-wallet.md [hunk 29]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Your Golem wallet and Yagna setup for Mainnet payments.
-title: Golem Wallet management for requestors
+title: Managing Your Golem Wallet and Yagna for Mainnet 
+description: Guide to managing Golem wallet and Yagna for receiving and sending mainnet payments, configuring networks, and generating app-keys.
```
Comment (jalas167) [inline src/pages/docs/creators/python/examples/tools/managing-golem-wallet.md:3]:
```suggestion
description: Instructions on how to manage Golem wallet and Yagna for receiving and sending mainnet payments, configure networks, and generate app-keys. 
```

File: src/pages/docs/creators/python/examples/tools/managing-golem-wallet.md [hunk 30]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Your Golem wallet and Yagna setup for Mainnet payments.
-title: Golem Wallet management for requestors
+title: Managing Your Golem Wallet and Yagna for Mainnet 
+description: Guide to managing Golem wallet and Yagna for receiving and sending mainnet payments, configuring networks, and generating app-keys. 
+pageTitle: Golem Wallet & Yagna Mainnet Management - Payment Setup Guide
```
Comment (jalas167) [inline src/pages/docs/creators/python/examples/tools/managing-golem-wallet.md:4]:
```suggestion
pageTitle: Golem Wallet & Yagna Mainnet Management - Payment Setup Instructions
```

File: src/pages/docs/creators/python/examples/tools/publishing-custom-images.md [hunk 31]
```diff
@@ -1,6 +1,7 @@
 ---
-title: Publishing an image
-description: Guide on how to publish a Golem image to the registry using gvmkit-build
+title: Guide to Publishing Golem Images on the Registry
```
Comment (jalas167) [inline src/pages/docs/creators/python/examples/tools/publishing-custom-images.md:2]:
```suggestion
title: Instructions on how to Publish Golem Images on the Registry 
```

File: src/pages/docs/creators/python/examples/tools/restoring-golem-wallet.md [hunk 32]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Setting up a new Yagna instance to use a saved keystore file.
-title: Restoring a Golem wallet from a keystore backup.
+title: Wallet Recovery via Keystore in Yagna 
+description: Learn how to restore your Golem wallet using a keystore file with this comprehensive guide on wallet recovery in Yagna.
```
Comment (jalas167) [inline src/pages/docs/creators/python/examples/tools/restoring-golem-wallet.md:3]:
```suggestion
description: Learn how to restore your Golem wallet using a keystore file with this comprehensive instruction focused on wallet recovery in Yagna. 
```

File: src/pages/docs/creators/python/examples/tools/restoring-golem-wallet.md [hunk 33]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Setting up a new Yagna instance to use a saved keystore file.
-title: Restoring a Golem wallet from a keystore backup.
+title: Wallet Recovery via Keystore in Yagna 
+description: Learn how to restore your Golem wallet using a keystore file with this comprehensive guide on wallet recovery in Yagna. 
+pageTitle: Recover Your Golem Wallet - Keystore Restoration Guide for Yagna
```
Comment (jalas167) [inline src/pages/docs/creators/python/examples/tools/restoring-golem-wallet.md:4]:
```suggestion
pageTitle: Recover Your Golem Wallet - Keystore Restoration Instructions for Yagna
```

File: src/pages/docs/creators/python/examples/tools/self-hosted-vm-images.md [hunk 34]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Publishing VM images outside Golem's repository
-title: Publishing VM images outside Golem's repository
+title: Guide to Self-Hosting VM Images for Golem Developers 
+pageTitle: How to Self-Host and Distribute Golem VM Images – A Developer's Tutorial
```
Comment (jalas167) [inline src/pages/docs/creators/python/examples/tools/self-hosted-vm-images.md:3]:
```suggestion
pageTitle: How to Self-Host and Distribute Golem VM Images – concise instructions
```

File: src/pages/docs/creators/python/examples/tools/self-hosted-vm-images.md [hunk 35]
```diff
@@ -1,6 +1,7 @@
 ---
-description: Publishing VM images outside Golem's repository
-title: Publishing VM images outside Golem's repository
+title: Guide to Self-Hosting VM Images for Golem Developers
```
Comment (jalas167) [inline src/pages/docs/creators/python/examples/tools/self-hosted-vm-images.md:2]:
```suggestion
title: Examples of how to Self-Host VM Images
```

File: src/pages/docs/creators/python/examples/tools/yagna-installation-for-requestors.md [hunk 36]
```diff
@@ -1,6 +1,9 @@
 ---
-description: Yagna service installation
-title: Yagna installation for requestors
+title: Guide to Installing Yagna for Golem Network Interaction
```
Comment (jalas167) [inline src/pages/docs/creators/python/examples/tools/yagna-installation-for-requestors.md:2]:
```suggestion
title: Guide to Installing Yagna for Golem Network Interaction 
```
```suggestion
title: Instructions for Installing Yagna for Golem Network Interaction 
```

File: src/pages/docs/creators/python/examples/tools/yagna-installation-for-requestors.md [hunk 37]
```diff
@@ -1,6 +1,9 @@
 ---
-description: Yagna service installation
-title: Yagna installation for requestors
+title: Guide to Installing Yagna for Golem Network Interaction 
+pageTitle: Install Yagna - Connect with the Golem Network - Requestor Setup Guide
```
Comment (jalas167) [inline src/pages/docs/creators/python/examples/tools/yagna-installation-for-requestors.md:3]:
```suggestion
pageTitle: Install Yagna - Connect with the Golem Network - Requestor Setup Instructions 
```

File: src/pages/docs/troubleshooting/javascript/index.md [hunk 38]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Javascript troubleshooting
-description: This is the troubleshooting section for all javascript related errors on Golem.
+title: Golem JavaScript SDK Troubleshooting Guide
```
Comment (jalas167) [inline src/pages/docs/troubleshooting/javascript/index.md:2]:
```suggestion
title: Golem JavaScript SDK Troubleshooting 
```

File: src/pages/docs/troubleshooting/js-requestor/exit-codes.md [hunk 39]
```diff
@@ -1,6 +1,7 @@
 ---
-title: Requestor exit codes
-description: Find all exit codes that may occur as a requestor on the golem network
+title: Golem Requestor Troubleshooting - Exit Code Meanings 
+description: Comprehensive guide to understanding Linux exit codes encountered as a requestor on the Golem Network for effective debugging.
```
Comment (jalas167) [inline src/pages/docs/troubleshooting/js-requestor/exit-codes.md:3]:
```suggestion
description: Resources to understand Linux exit codes encountered as a requestor on the Golem Network for effective debugging. 
```

File: src/pages/docs/creators/python/examples/tools/restoring-golem-wallet.md (hunk 32 above)
Comment (cryptobench) [reply src/pages/docs/creators/python/examples/tools/restoring-golem-wallet.md:3]: I don't think it sounds right in this one.

**Review by jalas167 — APPROVED** [review]

**Pull request merged**